src/interpreter/ByteCode.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Walrus.h"
+
+#include "interpreter/ByteCode.h"
+
+namespace Walrus {
+
+size_t ByteCode::getSize()
+{
+    OpcodeKind opcode = this->opcode();
+
+    switch (opcode) {
```

Comment:
Add an assert before: `g_byteCodeInfo[opcode].m_size > 0`. This way unused opcodes cannot be used as byte codes.

Reply:
It should be implemented.